toolchain/lc-compile/src/report.c

```diff
@@ -237,6 +237,8 @@ DEFINE_ERROR_I(UnableToFindImportedModule, "Unable to find imported module '%s'"
 DEFINE_ERROR_S(MalformedToken, "Illegal token '%s'");
 DEFINE_ERROR_S(MalformedEscapedString, "Illegal escape in string '%s'");
 DEFINE_ERROR(MalformedSyntax, "Syntax error");
+DEFINE_ERROR(InvalidIntegerLiteral, "Malformed integer literal");
```

Comment:
Definitely out of the scope of this PR, but it would be quite nice to have an explicit "Integer literal too large" error message that's distinct from "Malformed integer literal" since they're not quite the same thing.